src/audio/aria/aria.toml

```diff
@@ -0,0 +1,25 @@
+	// # Aria module config
+	[[module.entry]]
+	name = "ARIA"
+	uuid = "99F7166D-372C-43EF-81F6-22007AA15F03"
+	affinity_mask = "0x1"
+	instance_count = "8"
```

Comment:
Before we merge this, we should discuss future of fields like "affinity-mask" and "instance-count" that are not really related to the module definition, but rather to the product/configuration it's used on (affinity mask depends on number of cores the system has, and instance count will depend on available memory on the system).

I think we can agree there is just one value in SOF for these values (not able to variate between configurations), and proceed with merge, but worth to stop for a moment before merge.

For docs, see https://github.com/thesofproject/sof-docs/pull/451/files (itself stll not merged)

Reply:
Good point. Shall we do
```
	affinity_mask = ARIA_AFFINITY_MASK
	instance_count = ARIA_INSTANCE_COUNT
```
in aria.toml and
```
define those in each platform.toml.h
```
#ifdef CONFIG_COMP_ARIA
#define ARIA_AFFINITY_MASK "0x1"
#define ARIA_INSTANCE_COUNT "8"
#include <audio/aria/aria.toml>
#endif
```
?

Reply:
Lets leave these as a TODO for next updates/improvement cycle.

Reply:
Yes! Please centralize the config!